src/collections/members/adithya-krishna/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+name: Adithya Krishna
+position: Contributor
+image_path: ./adithya-krishna.jpg
+github: adithyaakrishna
+twitter: adii_kris
+linkedin: adiiikris
+location: Bengaluru, Karnataka, IN
+bio: An enthusiastic Final Year CSE student at Jyothy Institute of Technology, Bengaluru and trying to explore newer fields and a person who loves to contribute to Open Source Software. I'm also the type of person who says Yes! to everything spontaneously as I believe in utilizing every opportunity I can get and learning/upskilling myself in the mean process 👨‍💻
+blurb: Mesh Head
```

Comment:
@Tanuj22 I guess there was this line to be added right, which was according to the content on the issue at #1804 

![image](https://user-images.githubusercontent.com/23498248/116597257-05320800-a943-11eb-9426-fba2b210ffcd.png)

Reply:
Yes correct. @OlaoluwaM can you add it

Reply:
Sure!